scripts/build.ps1

```diff
@@ -167,6 +167,14 @@ param (
 Set-StrictMode -Version 'Latest'
 $PSDefaultParameterValues['*:ErrorAction'] = 'Stop'
 
+if ($Parallel -lt -1) {
+    if ($IsWindows) {
+        $Parallel = -1
```

Comment:
Does `$Parallel` being -1 mean that CMake will use the platform's level of parallelism?

Reply:
-1 means no parallelization. 0 means platform level (That happens down around line 345)

Reply:
And in this case, we want no parallelization on Windows (/MP to the compiler handles that) and we want platform level of parallelization on unix platforms so it will use all cores by default.